_pages/develop/context-browser.md

```diff
@@ -372,6 +372,68 @@ The following CodePen shows this all working together:
+* `favoritedSyncing: boolean = false` - If true, a loading indicator will be shown to indicate that the app is attempting to resync locally favorited items with those saved on the server. If false, the loading indicator will be hidden.
+
+The following events are fired to help with favoriting and de-favoriting:
+
+* `px-app-asset-favorited` - Fired when a new item is favorited. Includes details about how the item was favorited, and information about the new favorited item.
+* `px-app-asset-defavorited` - Fired when a previously favorited item is de-favorited. Includes details about how the item was de-favorited, and information about the de-favorited item.
+* `px-app-asset-favorited-sync-requested` - Fired when the user manually re-syncs their local favorites with those stored on their server.
+
+The following methods can be used to favorite and de-favorite items:
+
+* `favorite(item: Object|Array<Object>, source: String)` - Call with an object that is a direct reference to one of the `items` objects to favorite a single item, or with an array of objects to favorite multiple items.
+* `defavorite(item: Object|Array<Object>|null, source: String)` - De-favorites one or more items. Call with a direct reference to one of the favorited items to de-favorite it, or an array of favorited items to de-favorite multiple, or `null` to de-favorite all items.
+
+
+## Favoriting and selecting favorited items
+
+When the user hovers over an item in the context browser and favoriting is enabled, they will be presented with a star icon they can tap to add that item to their list of favorites. To de-favorite an item, the user can tap the star icon again.
+
+To see all favorited items, the user can open the favorites menu by tapping the favorites trigger. This will open the user's list of favorites, from which they can de-favorite an item or select it to open its context.
+
+Selecting an item from the Favorited Panel works the same as selecting an item from the regular context browser: clicking on an item will open that item's context, and the user will see a visual indication of the currently selected item when they re-open either the Favorited Panel or the regular context browser.
+
+## De-favoriting items
+
+The user can de-favorite an item from the favorited list by clicking on the star icon. De-favoriting an item will remove it from the favorited list, but the de-favoriting will not be reflected in the list until the user either closes the Favorited Panel for more than 5 seconds, or opens the regular Context Browser panel. In the interim, the item will remain in the favorited list, but the star icon indicating that it is favorited will not be filled in. Having a 5 second delay before an item is de-favorited allows the user to reopen the Favorited Panel and still see the item in the list (and re-favorite it) if the de-favoriting was done in error. De-favoriting an item from the Favorited Panel will be reflected in the Context Browser as well.
```

Comment:
While it's not the dictionary, most of the internet seems to agree that "unfavorite" is the appropriate opposite of favorite.

Reply:
Although I see now that this would be a code change as well.

Reply:
Yeah, we discussed this a bunch but decided to use "defavorite" to keep the API consistent with the activate and select APIs